File: Python/Slice/reverse_slice.py
# Reverse the slice order.
def reverse_slice(provided_slice):
    provided_slice.reverse()
    return provided_slice


def main():
    # Make a slice using a random set of data.
    randomSlice = ["a", "b", "c"]
    # Reverse the slice order.
    print(reverse_slice(randomSlice))


main()

File: TypeScript/Slice/reverse-slice.ts
function main(): void {
    // Create a new slice.
    var randomSlice: string[] = ["a", "b", "c"]
    // Reverse the slice, print the slice.
    console.log(reverseTheSlice(randomSlice))
}

main()


// Reverse the order of the values in the slice.
function reverseTheSlice(originalSlice: string[]): string[] {
    return originalSlice.reverse()
}